content/en/docs/components/pipelines/multi-user.md

```diff
@@ -167,19 +167,9 @@ without access control:
 * Artifacts, Executions, and other metadata entities in [Machine Learning Metadata (MLMD)](https://www.tensorflow.org/tfx/guide/mlmd).
 * [Minio artifact storage](https://min.io/) which contains pipeline runs' input/output artifacts.
 
-### In-cluster API request authentication
+## In-cluster API request authentication
 
-Clients can only access the Kubeflow Pipelines API from the public endpoint
```

Comment:
We probably want to keep some preamble like this (possibly reworded), about how the Istio auth is setup in Kubeflow.

For example, most users may not realize that the `istio-ingressgateway` is how Kubeflow APIs are exposed external to the clsuter.

Reply:
I searched the term "ingress gateway" on kubeflow.org and found that it's often mentioned in distribution-specific installation steps.

Istio ingress gateway is not being used exclusively by Kubeflow Pipelines. If we want to give more details about access etc. then something like [Traffic Flow](https://www.kubeflow.org/docs/distributions/aws/authentication/authentication/#traffic-flow) section could serve as a starting point and would probably fit better on [Overview](https://www.kubeflow.org/docs/started/kubeflow-overview/) page. If you agree, I could create separate issue+PR for that.